src/main/java/io/tus/java/client/TusClient.java

```diff
@@ -115,6 +116,39 @@ public boolean removeFingerprintOnSuccessEnabled() {
         return removeFingerprintOnSuccessEnabled;
     }
 
+    /**
+     * Enable chunked transfer encoding.
+     *
+     * @see https://en.wikipedia.org/wiki/Chunked_transfer_encoding
+     * @see #disableChunkedTransferEncoding()
+     */
+    public void enableChunkedTransferEncoding() {
+        chunkedTransferEncoding = true;
+    }
+
+    /**
+     * Disable chunked transfer encoding.
+     *
+     * @see https://en.wikipedia.org/wiki/Chunked_transfer_encoding
+     * @see #enableChunkedTransferEncoding()
+     */
+    public void disableChunkedTransferEncoding() {
+        chunkedTransferEncoding = false;
+    }
+
+    /**
+     * Get the current status of chunked transfer encoding.
+     *
+     * @see https://en.wikipedia.org/wiki/Chunked_transfer_encoding
+     * @see #enableChunkedTransferEncoding()
+     * @see #disableChunkedTransferEncoding()
+     *
+     * @return True if chunked transfer encoding has been enabled.
+     */
+    public boolean chunkedTransferEncoding() {
```

Comment:
Please rename this function to `chunkedTransferEncodingEnabled` so it's similar to the other method names.